bundles/org.openhab.binding.opengarage/src/main/resources/ESH-INF/i18n/opengarage_xx_XX.properties

```diff
@@ -0,0 +1,17 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
```

Comment:
remove if you don't provide i18n

Reply:
I believe this is addressed in 4714fc9b7d